src/utils/cli-utils.ts

```diff
@@ -1,7 +1,12 @@
 import * as edgeWorkersClientSvc from '../edgeworkers/client-manager';
 import * as envUtils from '../utils/env-utils';
+
 const inquirer = require('inquirer');
 const Spinner = require('cli-spinner').Spinner;
+const TIME_UNITS: string[] = ['ms', 's', 'min', 'h'];
+const MEMORY_UNITS: string[] = ['B', 'kB', 'MB', 'GB', 'TB'];
+const COUNT_UNITS: string[] = ['', 'Thousand', 'Million', 'Billion', 'Trillion'];
```

Comment:
what does empty string mean here ?

Reply:
Empty string is the count limit unit for when the count value is under 1000 (i.e 1-999 count will have the empty string, and 1000 would be displayed as 1 Thousand)